src/SafientCore.ts

```diff
@@ -351,15 +353,16 @@ export class SafientCore {
           const safe: string[] = await createSafe(data);
   
           if (onChain === true) {
-            const metaDataEvidenceUri: string = await createMetaData(
-              '0xe7f1725E7734CE288F8367e1Bb143E90bb3F0512',
-              creatorUser[0].userAddress
-            );
+            
   
             const arbitrationFee: number = await this.arbitrator.getArbitrationFee();
```

Comment:
We can move this inside the if condition of arbitrationbased claim. After line 365